src/nomos/agent_tests/testdata/LastGoodNomosTestfilesScan

```diff
@@ -166,8 +166,8 @@ File NomosTestfiles/LGPL/fsp-parser.h contains license(s) LGPL-3.0
 File NomosTestfiles/LGPL/LGPL-2.1.txt contains license(s) LGPL-2.1
 File NomosTestfiles/LGPL/GPL_false-positive_x_bug2307 contains license(s) LGPL-2.1+,MIT-CMU-style
 File NomosTestfiles/LGPL/LGPL-2.1b.txt contains license(s) LGPL-2.1
-File NomosTestfiles/LGPL/openjdk-7-jre-headless_license.txt contains license(s) W3C,AGPL-3.0+,ATT,Apache-1.1-style,Apache-2.0,Freeware,GPL-2.0+-with-classpath-exception,ICU,IJG,LGPL-2.0,NPL-1.1,NotreDame-style,SAX-PD,Unicode,XFree86
-File NomosTestfiles/LGPL/defconfig contains license(s) BSD,LGPL
+File NomosTestfiles/LGPL/openjdk-7-jre-headless_license.txt contains license(s) Apache-2.0,W3C,AGPL-3.0+,ATT,Apache-1.1-style,Freeware,ICU,IJG,LGPL-2.0,NPL-1.1,NotreDame-style,OpenJDK-assembly-exception-1.0,SAX-PD,Unicode,XFree86
+File NomosTestfiles/LGPL/defconfig contains license(s) GPL,BSD
```

Comment:
nice! it actually got better here:

https://github.com/fossology/fossology/blob/master/src/nomos/agent_tests/testdata/NomosTestfiles/LGPL/defconfig